### ISSUE TITLE
Finish History Documentation/API

### ISSUE THREAD

**Issue by awkay** [fulcrologic/fulcro#287]
History is a somewhat overlooked feature.  Inspect interacts with it (in a way), but as a first-class library feature it is somewhat "lacking".  The most important use, the support viewer, is rarely used I think, probably because it requires a bit of setup.

Goals of this task are:
- [ ] Make the support viewer super easy to set up and use
- [ ] Use something like rendering-to-an-iframe to ensure the CSS/look of the app under inspection
- [ ] Add docs to explain how to set up infrastructure and debug against application history
- [ ] Talk about including app version information with the app history

Ultimately, show how the history viewer enables developers to easily find/fix bugs in application code based on user error reports.

**Comment by adamfeldman** [comment]
I absolutely love the support viewer. To me it's a selling point of the framework as a whole, as it demonstrates how _Fulcro feels highly considerate to the practical needs of real-world complex products_.

I think the enhancement goals are excellent. In particular, thank you for your usual attention to docs. I am excited about documented recommendations for the infrastructure pieces that allow the viewer to be used in practice.

(Tiny) experience report: I am a developer turned product manager turned founder. One of the first things I did as I started playing with Clojure was get the [fulcro-todomvc](https://github.com/fulcrologic/fulcro-todomvc) demo running, so that I could try out the support viewer. It was fairly straightforward, despite my inexperience with Clojure tooling.

The vision for the viewer reminds me of a useful commercial product that is a developer-focused web session playback tool called  LogRocket (which is also comparable to the the well-known FullStory). When I get closer to launching my Fulcro app, I will consider integrating the support viewer with LogRocket in a way similar to how [LogRocket integrates with Redux](https://docs.logrocket.com/reference).
